app/Actions/InitializeGitHubRepository.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Actions;
+
+use App\ConsoleWriter;
+use App\Shell;
+
+class InitializeGitHubRepository
+{
+    use AbortsCommands;
+
+    public const WARNING_FAILED_TO_CREATE_REPOSITORY = 'Failed to create new GitHub repository';
+
+    protected $shell;
+    protected $consoleWriter;
+
+    public function __construct(Shell $shell, ConsoleWriter $consoleWriter)
+    {
+        $this->shell = $shell;
+        $this->consoleWriter = $consoleWriter;
+    }
+
+    public function __invoke()
+    {
+        $ghCommandOptions = config('lambo.store.github');
```

Comment:
Do we need to perhaps validate the parameters they're passing here? could it possibly be `true` if they just send in `--github`?

Reply:
Updated to follow the behaviour of the Laravel installer. Now, if `--github` is passed, it is the equivalent of passing `--github='--private'`.

In essence `--github` is a shortcut for creating a private repository with issue tracking.